march_joint_inertia_controller/include/march_joint_inertia_controller/joint_inertia_controller.h

```diff
@@ -3,23 +3,76 @@
 #ifndef MARCH_HARDWARE_JOINT_INERTIA_CONTROLLER_H
 #define MARCH_HARDWARE_JOINT_INERTIA_CONTROLLER_H
 
+#include <control_toolbox/pid.h>
 #include <controller_interface/controller.h>
-#include <hardware_interface/joint_command_interface.h>
-#include <pluginlib/class_list_macros.h>
+#include "march_joint_inertia_controller/inertia_estimator.h"
+#include <std_msgs/Float64.h>
 
 namespace joint_inertia_controller
 {
-class InertiaController : public controller_interface::Controller<hardware_interface::PositionJointInterface>
+class InertiaController : public controller_interface::Controller<hardware_interface::EffortJointInterface>
 {
 public:
+  struct Commands
+  {
+    double position;    // Last commanded position
+    double velocity;    // Last commanded velocity
+    bool has_velocity;  // false if no velocity command has been specified
+  };
+
+  InertiaController() = default;
+  ~InertiaController() override;
+
   bool init(hardware_interface::PositionJointInterface* hw, ros::NodeHandle& n);
```

Comment:
I think this should fix it so that you can override.
```suggestion
  bool init(hardware_interface::EffortJointInterface* hw, ros::NodeHandle& n) override;
```